### PR TITLE
variablize solidty and assertions to avoid naming conflicts

### PR DESCRIPTION
**Pull request by iamchrissmith**
Doing some brute find and replace to eliminate variable naming conflicts in the solidity tests.

Current example output:

```
### Solidity
#### ID:2
------------

    // Test Run
    admin.vat_rely(address(goldJoin));
    admin.spotter_setPrice("gold", 1);
    admin.goldFlip_rely(address(end));
    admin.vat_file("Line", 1000000000000);
    admin.vat_file("spot", "gold", 3000000000);
    admin.vat_file("line", "gold", 1000000000000);
    admin.vow_file("bump", 1000000000);
    admin.vow_file("hump", 0);
    admin.Gem_gold_mint(address(alice), 20);
    admin.Gem_gold_mint(address(bobby), 20);
    alice.vat_hope(address(pot));
    alice.vat_hope(address(goldFlip));
    alice.vat_hope(address(end));
    bobby.vat_hope(address(pot));
    bobby.vat_hope(address(goldFlip));
    bobby.vat_hope(address(end));
    alice.goldJoin_join(address(alice), 10);
    bobby.goldJoin_join(address(bobby), 10);
    alice.vat_frob("gold", address(alice), address(alice), address(alice), 10, 10);
    bobby.vat_frob("gold", address(bobby), address(bobby), address(bobby), 10, 10);
    alice.vat_move(address(alice), address(alice), 575_/Rat_64);
    admin.pot_file("dsr", 1489_/Rat_1280);
    hevm.warp(2);
    alice.vat_hope(address(alice));
    hevm.warp(1);
    hevm.warp(1);

    // Assertions
    assertTrue( pot.dsr() == 1489 /Rat 1280 );
    assertTrue( vat.can() == "bobby" |-> SetItem( Flip "gold" )
SetItem( pot )
SetItem( end )
"alice" |-> SetItem( Flip "gold" )
SetItem( "alice" )
SetItem( pot )
SetItem( end ) );

------------
### /Solidity
```

I didn't mess too much with the `assert` section since we still need a bigger PR to deal with the `SetItem` and other mapping state changes.

### CODE REVIEW

**Comment by gbalabasquer** [comment]
The output looks good, just `Gem_gold_mint` could be replaced to something else to keep consistency.
Ideal would be `gold_mint` or `gemGold_mint`.
Is there any case we will get exported the `gold` token itself. I'm doubting it might enter in some conflict with the "gold" `ilk` string processing.

**Comment by gbalabasquer** [comment]
`Gem_MKR` => `gov`

**Comment by gbalabasquer** [comment]
Just noticed `vat_file("line", "gold", 1000000000000);` is `what,ilk,data`. When the contract order is `ilk,what,data` I can handle that in the solidity side, but wanna double check that always will come this way.

File: mcd-pyk.py [hunk 1]
```diff
@@ -220,14 +220,31 @@ def _gatherViolations(fsmMap):
...
+    return input.replace('Alice', 'alice')          \
+                .replace('Bobby', 'bobby')          \
+                .replace('ADMIN', 'admin')          \
+                .replace('ANYONE', 'anyone')        \
+                .replace('Vat', 'vat')              \
+                .replace('Vow', 'vow')              \
+                .replace('Cat', 'cat')              \
+                .replace('Pot', 'pot')              \
+                .replace('Flap', 'flap')             \
+                .replace('Flop', 'flop')             \
```
Comment (ehildenb) [inline mcd-pyk.py:233]:
```suggestion
                .replace('Flap', 'flap')            \
                .replace('Flop', 'flop')            \
```

**Pull request merged**